### PR TITLE
Single image

### PR DESCRIPTION
**Pull request by Nischhal3**

### CODE REVIEW

**Review by thuymymai — APPROVED** [review]
did you delete all console logs?

**Comment by GiangNguyen1207** [comment]
no he didn't

File: server/models/imageModel.js [hunk 1]
```diff
@@ -91,18 +92,21 @@ const deleteImage = async (imageId, user_id, role, next) => {
 };
 
 const updateImage = async (user_id, image, next) => {
-  let sql =
-    'UPDATE image_db SET image_title = ?, image_description = ? WHERE image_id = ? AND user_id = ? ';
-  let params = [image.image_title, image.image_description, image.id, user_id];
+  console.log('update', user_id, image);
```
Comment (GiangNguyen1207) [inline server/models/imageModel.js:91]: remove console.log please :D

File: server/controllers/imageController.js [hunk 2]
```diff
@@ -75,21 +75,18 @@ const post_image = async (req, res, next) => {
 };
 
 const delete_image = async (req, res, next) => {
-  const image_id = req.params.imageId;
+  const image_id = req.params.id;
   const user_id = req.user.user_id;
   const role = req.user.role;
-  console.log('Image id', image_id);
+  console.log('Image id', image_id, user_id);
```
Comment (GiangNguyen1207) [inline server/controllers/imageController.js]: this one also

File: client/pages/collection/singleImage.js [hunk 3]
```diff
@@ -26,16 +27,18 @@ const getImage = async (id) => {
     const image = await response.json();
     createPath(image.collection_id, image.collection_title, image.image_title);
     createImageCard(image);
-    console.log(image);
+    console.log('image', image);
```
Comment (GiangNguyen1207) [inline client/pages/collection/singleImage.js]: this one also

**Review by GiangNguyen1207 — CHANGES_REQUESTED** [review]

File: client/pages/collection/singleImage.js (hunk 3 above)
Comment (thuymymai) [reply client/pages/collection/singleImage.js]: I deleted  the console logs :D

File: server/controllers/imageController.js [hunk 4]
```diff
@@ -78,9 +78,11 @@ const delete_image = async (req, res, next) => {
   const image_id = req.params.id;
   const user_id = req.user.user_id;
   const role = req.user.role;
-  console.log('Image id', image_id, user_id);
-  console.log('Id ', role);
   const deleted = await deleteImage(image_id, user_id, role, next);
+  if(!deleted){
+    res.json({ message: `Image deleted ${deleted}. Cannot delete others' image.` });
```
Comment (GiangNguyen1207) [inline server/controllers/imageController.js]: image deleted and can not delete others image? lol

File: server/controllers/imageController.js [hunk 5]
```diff
@@ -97,6 +99,10 @@ const update_image = async (req, res, next) => {
   }
 
   const update = await updateImage(user_id, req.body, next);
+  if(!update){
+    res.json({ message: `Image update ${update}. Cannot update others' image.` });
```
Comment (GiangNguyen1207) [inline server/controllers/imageController.js]: same here lmao

File: server/controllers/imageController.js (hunk 4 above)
Comment (thuymymai) [reply server/controllers/imageController.js]: image delete false ma ma

**Pull request merged**